script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -1180,8 +1180,9 @@ We're already 100% certain that Tohno Shiki is Roa!"
 [sha:3b49729e73aecf4c7f4bacaa207f3d96c24366cb]{
 -- Page 741, Offset 26290.
 -- 「100パーセントだあ？　なら前提が間違えてんだよ、このお布施泥棒が。オレは悪名高き“人間使い”、ラウレンティス枢機卿の秘蔵っ子だぞ？　人を見る目だけはテメェにも、テメェの師匠にも負けてねえ」
+//人間使い is Mario's nickname, and is playing off of 人形使い (Puppetmaster), so here I've rendered it as Peoplemaster to pun off of this.
```

Comment:
I talked to a friend about this one and his suggestion was "thrallmaster" which I think doesn't sound so bad

Reply:
If its between that and Peoplemaster I'd stick to Peoplemaster.